mainscript.py

```diff
@@ -153,10 +153,11 @@ def initialise():
 
     device_id = int(rawinput()) - 1
 
-    if not os.path.exists('Music'):
-        os.makedirs('Music')
+    music_dir = rawinput('What folder should the music be downloaded to?')
+    if not os.path.isdir(os.path.join(os.getcwd(), music_dir)):
+        os.makedirs(music_dir)
     click.secho(
-        "The music would be downloaded to {0}/Music".format(os.getcwd()))
+        "The music will be downloaded to {}/{}".format(os.getcwd(), music_dir)
```

Comment:
`{}/{}` shouldn't be used for printing out, since it wouldn't be right for windows. The whole path for the music directory should be printed.